scripts/install/opt/seagate/cortx/motr/libexec/m0reportbug-bundler

```diff
@@ -62,14 +71,28 @@ fi
 bundle_id=${1:-$HOSTNAME}
 dest_dir=${2:-$DEFAULT_DEST_DIR}
 
-while getopts ":b:t:c:s:" flag
-do
-    case "${flag}" in
-          b) bundle_id="$OPTARG";;
-          t) dest_dir="$OPTARG";;
-          c) config_url="$OPTARG";;
-          s) sevice="$OPTARG";;
-    esac
+duration=P5d
+size_limit=0
+services=All
+stacktrace=false
+coredumps=false
+binlogs=true
+TEMP=$(getopt -o c: --long duration:,size_limit:,services:,binlogs:,coredumps:,stacktrace: \
+                  -n "${0##*/}" -- "$@")
+eval set -- "$TEMP"
+while true; do
+     case "$1" in
+         -c) ;;
+         --duration) ;;
```

Comment:
Are you sure, we don't need to use shift here ?

Reply:
we need, but scope of this ticket to get command line option passed by support bundle generation command line:

Reply:
incorporated